athena/athena/athena/schemas/programming_feedback.py

```
@@ -1,30 +1,30 @@
-    line_start: Optional[int] = Field(None, description="Start line number, 0-indexed", ge=0, example=1)
-    line_end: Optional[int] = Field(None, description="End line number, 0-indexed", ge=0, example=2)
+    line_start: Optional[int] = Field(None, description="Start line number, 0-indexed", ge=0, examples=[1])
+    line_end: Optional[int] = Field(None, description="End line number, 0-indexed", ge=0, examples=[2])
 
-    @validator('line_start')
-    def validate_line_start(cls, v, values, **kwargs):
-        if 'line_end' in values and v is None and values['line_end'] is not None:
+    @field_validator('line_start')
+    def validate_line_start(cls, v: int, info: ValidationInfo):
+        if 'line_end' in info.data and v is None and info.data['line_end'] is not None:
             raise ValueError('line_start can only be None if line_end is None.')
```

Ruff (0.13.3)

Avoid specifying long messages outside the exception class

(TRY003)